src/dataprotection/azext_dataprotection/tests/latest/test_dataprotection_aks_workload.py

```diff
@@ -0,0 +1,231 @@
+                            '--source-datastore OperationalStore '
+                            '--recovery-point-id "{rp_id}" '
+                            '--backup-instance-id "{backup_instance_id}" '
+                            '--restore-configuration "{restore_config_json}"').get_output_in_json()
+    test.kwargs.update({"restore_request": restore_json})
+
+    # uncomment when running live, run only in record mode - grant permission
+    # test.cmd('az dataprotection backup-instance update-msi-permissions '
+    #          '--datasource-type AzureKubernetesService '
+    #          '--operation Restore '
+    #          '--permissions-scope Resource '
+    #          '--vault-name "{vaultname}" '
+    #          '--resource-group "{rgname}" '
+    #          '--restore-request-object "{restore_request}" '
+    #          '--snapshot-resource-group-id "{rg_id}" -y')
+    # time.sleep(60)
+
+    # Also uncomment when running live, only run in record mode - provide trusted access
+    # helper_trusted_access(test)
+
+    test.cmd('az dataprotection backup-instance validate-for-restore -g "{rgname}" --vault-name "{vaultname}" -n "{backup_instance_name}" --restore-request-object "{restore_request}"')
+
+    response_json = test.cmd('az dataprotection backup-instance restore trigger '
+                             '-g "{rgname}" '
+                             '--vault-name "{vaultname}" '
+                             ' -n "{backup_instance_name}" '
+                             '--restore-request-object "{restore_request}"').get_output_in_json()
+    job_status = None
+    test.kwargs.update({"backup_job_id": response_json["jobId"]})
+    while job_status != "Completed":
+        time.sleep(10)
+        job_response = test.cmd('az dataprotection job show --ids "{backup_job_id}"').get_output_in_json()
+        job_status = job_response["properties"]["status"]
+        if job_status not in ["Completed", "InProgress"]:
+            raise Exception("Undefined job status received")
+
+def delete_backup(test):
+    test.cmd('az dataprotection backup-instance delete -g "{rgname}" --vault-name "{vaultname}" -n "{backup_instance_name}" --yes')
+
+def cleanup(test):
+    delete_backup(test)
+    test.cmd('az dataprotection backup-vault delete -g "{rgname}" --vault-name "{vaultname}" --yes')
+
+    try:
+        test.cmd('az aks trustedaccess rolebinding delete -g "{rgname}" --cluster-name "{akscluster1}" -n aksRoleBindingName --yes')    
+    except:
+        # This is usually failing in the event of the trusted access already not existing, so we can skip it unless a new error shows.
+        pass
+
+@AllowLargeResponse()
+def call_scenario(test):
+    setup(test)
+    try:
+        setup_vault_and_policy(test)
+        configure_backup(test)
+        trigger_backup(test)
+        trigger_restore_original_location(test)
+    except Exception as e:
+        raise e
+    finally:
+        cleanup(test)
```

Comment:
Why not move them under the test_dataprotection_aks?

Reply:
In this file I am following the pattern that we already have in `test_dataprotection_oss_workload.py`